pkg/request.go

```diff
@@ -66,7 +66,7 @@ func unmarshalRequest(jsonRequest []byte) (*request, error) {
 
 	// Validate
 	if !validMethod(req.Method) {
-		return req, fmt.Errorf("invalid method: %s", req.Method)
+		return nil, fmt.Errorf("invalid method: %s", req.Method)
```

Comment:
Why return `nil` instead of `req` here but not in https://github.com/loveholidays/ripley/blob/c608cb60c077158cc2d728bd0d5316b823367a7c/pkg/request.go#L73 or https://github.com/loveholidays/ripley/blob/c608cb60c077158cc2d728bd0d5316b823367a7c/pkg/request.go#L77? 

I don't think it matters, just looks a bit weird if they're different. I say return `nil` in all places, @npalumbo